app/assets/stylesheets/tripNew.scss

```diff
@@ -0,0 +1 @@
+@import "variables";
```

Comment:
if there are no page specific styles, this page can be deleted since there are no styles